website/docs/Capabilities.md

```diff
@@ -240,49 +240,136 @@ When testing on Chrome, WebdriverIO will automatically download the desired brow
 ```ts
 {
     browserName: 'chrome',
-    browserVersion: '116' // or '116.0.5845.96', 'stable', 'dev', 'canary', 'beta'
+    browserVersion: '116.0.5845.96' // or 'stable', 'latest', 'dev', 'canary', 'beta'
 }
 ```
 
+If you like to test a manually downloaded browser, you can provide a binary path to the browser via:
+
+```ts
+{
+    browserName: 'chrome',
+    'goog:chromeOptions': {
+        binary: '/Applications/Google\ Chrome\ Canary.app/Contents/MacOS/Google\ Chrome\ Canary'
+    }
+}
+```
+
+Additionally, if you like to use a manually downloaded driver, you can provide a binary path to the driver via:
+
+```ts
+{
+    browserName: 'chrome',
+    'wdio:chromedriverOptions': {
+        binary: '/path/to/chromdriver'
+    }
+}
+```
+
+:::caution
+
+If you set a driver binary, the browser won't be downloaded automatically. You'll need to have the corresponding browser already installed or specify a binary path as previously shown.
```

Comment:
Ref:

https://github.com/webdriverio/webdriverio/blob/aa35c291fb496b80350766d674385ea7b4d1516a/packages/wdio-utils/src/driver/manager.ts#L56-L57

and

https://github.com/webdriverio/webdriverio/blob/aa35c291fb496b80350766d674385ea7b4d1516a/packages/wdio-utils/src/driver/utils.ts#L161-L170

Applies for all other `:::caution` entries on this file.

Reply:
This mentioned before already when we document `binary` for within the common driver options:

> binary
> Path to a custom driver binary. If set WebdriverIO won't attempt to download a driver but will use the one provided by this path. Make sure the driver is compatible with the browser you are using.

I think repeating this for all browser again isn't ideal.

Reply:
Oh, I missed that! OK, I'm going to remove this in favor of promoting that to a `:::caution` so it gets more prominency.